src/engraving/libmscore/lyricsline.cpp

```diff
@@ -416,8 +416,14 @@ void LyricsLineSegment::layout()
     }
 
     // MELISMA vs. DASHES
+    static double minMelismaLen = 1 * sp; // TODO: style setting
```

Comment:
I think it shouldn't be static since `sp` isn't static either. But it could be const.

Reply:
Yeah, good point, it should be const. Though I think it can still be static: in this context it means that it is a property of the class, so it doesn't need to be reinstantiated by every lyricsLine object. Even if sp isn't static

Reply:
I still think it can't be static. `sp` may be different every time this method is called, so minMelismaLen must also be initialized/updated every time. Now, it will be initialized the first time this method is called, but after that, the value will never change anymore.

Reply:
Mmh, you're right. I was assuming that sp is always the same within a given score, but indeed it can change. Then the variable should just be a const double